src/en/charms-storage-ceph.md

```diff
@@ -0,0 +1,123 @@
+Title: Installing Ceph
```

Comment:
I would like to keep this rudimentary Ceph page as a sub-page to the storage page. We can definitely add a more sophisticated set of instructions elsewhere if anyone has time. I agree that  the Juju docs is not the place for in-depth treatments of third-party software. Not only is it inappropriate but it makes maintenance very heavy.